services/pull/pull.go

```diff
@@ -311,23 +311,39 @@ func checkIfPRContentChanged(pr *models.PullRequest, oldCommitID, newCommitID st
 func PushToBaseRepo(pr *models.PullRequest) (err error) {
 	log.Trace("PushToBaseRepo[%d]: pushing commits to base repo '%s'", pr.BaseRepoID, pr.GetGitRefName())
 
+	// Clone base repo.
+	tmpBasePath, err := models.CreateTemporaryPath("pull")
+	if err != nil {
+		log.Error("CreateTemporaryPath: %v", err)
+		return err
+	}
+	defer func() {
+		err := models.RemoveTemporaryPath(tmpBasePath)
+		if err != nil {
+			log.Error("Error whilst removing temporary path: %v", err)
```

Comment:
It would be nice to log the path to remove, so the admin can remove it themselves.

Reply:
```suggestion
			log.Error("Error whilst removing temporary path: %s Error: %v", tmpBasePath, err)
```

Reply:
It should literally never happen though.